Sources/Variable.swift

```diff
@@ -136,6 +118,24 @@ public struct Variable : Equatable, Resolvable {
   }
 }
 
+fileprivate func resolveCollection<T: Collection>(_ collection: T, bit: String) -> Any? {
```

Comment:
We're in the same type (`Variable`), so `private` is enough, `fileprivate` is for accessing from other types.